R/dualpathFused.R

```diff
@@ -10,7 +10,7 @@
 # the open interval to the *right* of the current lambda_k.
 
 dualpathFused <- function(y, D, approx=FALSE, maxsteps=2000, minlam=0,
-                          rtol=1e-7, btol=1e-7, verbose=FALSE,
+                          maxdf=length(y), rtol=1e-7, btol=1e-7, verbose=FALSE,
```

Comment:
I can't tell by eye but please make sure the column width doesn't go over 80 characters (that's the coding style with the rest of the package), and if so, just put the `verbose` argument on the next line.

Reply:
79 :)